### PR TITLE
FIPS Check Update

### PR DESCRIPTION
**Pull request by ejohnstown**
# Description

1. Whitespace changes.
2. Deleted the stm32l4 build. The fips-check-2.0.sh script handles it.
3. Removed the CAVP_SELFTEST_* variables, used the FIPS_OPTIONS for it.
4. Removed the v1 style code checkout. The current method works for all available cases.
5. Removed the FIPS_CONFLICTS case rename case.

# Testing

Kept a copy of the previous fips-check.sh script pre-byebyecyassl named orig-fips-check.sh. I wrote a script that called both orig-fips-check.sh and this version (which uses a different output directory), and then did a file compare between the wolfcrypt source and header files. I logged the output of the diff. The goal was to have absolutely no changes.

# Checklist

 - [ ] added tests
 - [ ] updated/added doxygen
 - [ ] updated appropriate READMEs
 - [ ] Updated manual and documentation

### CODE REVIEW

**Comment by ejohnstown** [comment]
This change is intended to make my next change easier. For each build, the files are going to be listed explicitly in tuples with the file revision to be checked out.

File: fips-check.sh [hunk 1]
```diff
@@ -36,152 +31,91 @@ Example:
...
-CRYPT_INC_PATH=undef
-CRYPT_SRC_PATH=undef
-RNG_VERSION=v3.6.0
-FIPS_OPTION=undef
-CAVP_SELFTEST_ONLY="no"
-GIT="git -c advice.detachedHead=false"
-
-if [ "$1" == "" ]; then FLAVOR="undef"; else FLAVOR="$1"; fi
+while [ "x$1" != 'x' ]; do
+  if [ "x$1" = 'xkeep' ]; then KEEP='yes'; else FLAVOR="$1"; fi
```
Comment (douzzer) [inline fips-check.sh:42]: `shellcheck` says
```
In fips-check.sh line 42:
  if [ "x$1" = 'xkeep' ]; then KEEP='yes'; else FLAVOR="$1"; fi
       ^---^ SC2268 (style): Avoid x-prefix in comparisons as it no longer serves a purpose.
```

**Review by douzzer — APPROVED** [review]

**Comment by ejohnstown** [comment]
Here's a link to my check script. https://gist.github.com/ejohnstown/0fee1429ef43bb515104e2894940a5f7

**Comment by ejohnstown** [comment]
Jenkins retest this

**Comment by ejohnstown** [comment]
retest this please

**Comment by ejohnstown** [comment]
retest this please

**Comment by douzzer** [comment]
retest this please.

**Comment by ejohnstown** [comment]
retest this please

**Review by dgarske — APPROVED** [review]

**Review by JacobBarthelmeh — APPROVED** [review]

**Pull request merged**